extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -75,29 +100,23 @@ public function updateImageState( $images, $action = '' ) {
 		$db->commit();
 
 		// update stats directly in memcache so they look nice without impacting the database
-		$key = wfMemcKey( 'ImageReviewSpecialController', 'ImageReviewHelper::getImageCount', $this->user_id );
-		$stats = $this->wg->memc->get($key, null);
-		if ($stats) {
-			switch ( $action ) {
-				case '':
-				//	$stats['reviewer'] += count($images);
-				//	$stats['unreviewed'] -= count($images);
-					$stats['questionable'] += count($sqlWhere[ImageReviewStatuses::STATE_QUESTIONABLE]);
-					break;
-				case ImageReviewSpecialController::ACTION_QUESTIONABLE:
-					$changedState = count( $sqlWhere[ImageReviewStatuses::STATE_APPROVED] ) + count( $sqlWhere[ImageReviewStatuses::STATE_REJECTED] );
-				//	$stats['reviewer'] += $changedState;
-					$stats['questionable'] -= $changedState;
-					break;
-				case ImageReviewSpecialController::ACTION_REJECTED:
-					$changedState = count( $sqlWhere[ImageReviewStatuses::STATE_APPROVED] ) + count( $sqlWhere[ImageReviewStatuses::STATE_DELETED] );
-					$stats['rejected'] -= $changedState;
-					break;
-			}
-			$this->wg->memc->set( $key, $stats, 3600 /* 1h */ );
-			// Quick hack/fix for stats going negative -- FIXME
-			if ($stats['unreviewed'] < 0)
-				$this->wg->memc->delete( $key );
+		switch ( $action ) {
```

Comment:
It will be nice if block like this put in separate method.
Also using sqlWhere is misleading a little.
also please use const in https://github.com/Wikia/app/pull/10879/files#diff-a56c48c5a45af2e5c50e11787482421fR62.